XamarinApp/XamarinApp/ViewModels/LoginPageViewModel.cs

```diff
@@ -0,0 +1,105 @@
+using NLog;
+using Prism.Commands;
+using Prism.Mvvm;
+using Prism.Navigation;
+using Prism.Services;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Xamarin.Forms;
+using XamarinApp.Models;
+using XamarinApp.Services.Interfaces;
+
+namespace XamarinApp.ViewModels
+{
+  public class LoginPageViewModel : ViewModelBase
+  {
+    private static Services.Interfaces.ILogger deplogger = DependencyService.Get<ILogManager>().GetLog();
```

Comment:
why is this not injected? also what's up with the naming of this?

Reply:
This is the old one which I have worked on following some documentation. But now I have replaced it by an Service Injection.